### DISCUSSION TITLE
Chinese Translation of the JSBSim User Manual

### DISCUSSION THREAD

**Discussion by 860128** [JSBSim-Team/jsbsim#1184] [Ideas]
I am a native Chinese-speaking developer and have noticed that the JSBSim user manual is only available in English. This presents some challenges for me and other non-native English speakers who may not be fluent in English. I am interested in translating the user manual into Chinese. 
Could you please let me know if this would be acceptable? How can I obtain permission to proceed with the translation? Also, where should I submit the translated manual once it is completed?

**Comment by agodemar** [comment]
@860128 Thank you for your interest in translating the manual. However, I personally believe that the guides and the documentation material should remain in English to keep the potential user audience as wide as possible. 
This does not mean, of course, that you cannot work on your own translation or that you cannot pass your translation on to others. The fact is that no one among us, the official developers and maintainers of JSBSim, can check and approve the translated manual in Chinese.
This is my opinion, of course.
@jonsberndt or @bcoconni or others could add more...

**Reply by yunzhongfei-z** [reply]
Hello professor, I am a beginner in JSBSim simulation, from Civil Aviation University of China, my name is Zhao Yunfei, my main goal is to achieve full autopilot flight from one airport to another, I have made some modifications based on C310-autopilot, combined with the script file, I can achieve the take-off and cruise functions of 737, but I cannot achieve functions such as automatic landing and CI index climb, which makes the aircraft unable to output normal simulation data. Therefore, I would like to ask you about the writing of the relevant autopilot, and hope to get your help, thank you. My 737 autopilot file is attached to the email.
